docs/en/components/esco-content-menu/README.md

```diff
@@ -87,6 +132,7 @@ For some integration you could need a bit more, like into uPortal you will need
 - `user-org-id-attribute-name`: type: `String`, default: `'ESCOSIRENCourant[0]'`, the attribute object path to obtain the id of the organization to retrieve from the organization's api
 - `user-all-orgs-id-attribute-name`: type: `String`, default: `'ESCOSIREN`, the attribute object path to obtain all ids of the organizations linked to the user and to retrieve from the organization's api
 - `org-logo-url-attribute-name`: type: `String`, default: `'otherAttributes.ESCOStructureLogo[0]'`, the attribute object path to obtain the organization Picture from organization details obtained from the organization's api
+- `force-org-logo`: type: `String`, optional, an url/uri to provide an institutional picture overriding the default-org-logo and the institutional one obtained by an api (used to have an overview before updating change to every users from the api),
```

Comment:
> used to have an overview before updating change to every users from the api

I'm not quite sure I follow this.
Are you saying this is used as a placeholder while the icon is retrieved from the API?

Reply:
You have the 'organizationApiUrl' property that permit to retrieve the organization information containing the organization logo.
With this property you provide a placeholder of a picture that will override it. This change will help to provide an overview of the new picture for the admin that is changing the organization logo in live and before to apply it from the API to all users. An other way would be to set this property as a Boolean to force the 'defaultOrgLogo' and setting the new placeholder to test on.

Reply:
So this is meant for testing purposes?

Reply:
Yes but in production use of the component and without to upgrade the rest API before to validate/apply